pkgs/tools/graphics/gifski/default.nix

```diff
@@ -2,7 +2,7 @@
 , rustPlatform
 , fetchFromGitHub
 , pkg-config
-, ffmpeg
+, ffmpeg_5
```

Comment:
overriding in all-packages is preferred https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults